system-configuration-sys/src/lib.rs

```diff
@@ -16,6 +16,12 @@
 #![allow(non_upper_case_globals)]
 #![allow(non_snake_case)]
 
-extern crate core_foundation_sys;
+pub extern crate core_foundation_sys;
+pub extern crate libc;
```

Comment:
not reexport `libc` is better ?

Use case:

Cargo.toml

```
[package]
name = "my-project"
version = "0.1.0"

[dependencies]
core-foundation-sys = "0.5"
libc = "10.10"       # Version Problem
```

that's maybe have problem if you reexport a diffence `libc` version,
that's maybe is why `env_logger` is not reexport `log`.
but i'm not sure.

Reply:
I do re-export it to get rid of version incompatibility problems. I can now use `system_configuration_sys::libc::c_void` (see https://github.com/mullvad/system-configuration-rs/blob/simplify-bindings/system-configuration/src/dynamic_store.rs#L24) in order to reach this type. Then above crates don't have to depend on `libc` themselves in order to use `system_configuration_sys`. This should get rid of incompatibility, because now a crate that uses this can depend on `libc = "10.10"` and use that freely, but it can also reach the `c_void` in `libc 0.2` because we re-export it. If we don't re-export `libc` that means a part of our public API is not reachable from our crate and that forces the user of `system_configuration_sys` to also depend on the exact same version of `libc` as us, now they don't have to.

Reply:
Your example with `libc = "10.10"` would not work if we don't re-export `libc`. Because then the `c_void` available in `my-project` would be different from the one required in the API of `system_configuration_sys`.

Reply:
Got it, thank you :)